ts/src/test/static/response/kucoin.json

```diff
@@ -1137,6 +1137,7 @@
         ],
         "fetchCurrencies": [
             {
+                "disabledCS": true,
```

Comment:
@ttodua why we need this?

Reply:
let me recheck, i don't remember the exact reason why i disabled that specific.

Reply:
have been checking for a while and eventually found the reason, so it's because this is set to empty dict: https://github.com/ccxt/ccxt/blob/9f27ec7245c5ecd9e6b3d61c027d1f659c287fb3/cs/ccxt/exchanges/kucoin.cs#L1246
while in other langs it's not empty.

that's the reason I had a PR that tried to transpiled

Reply:
so, the problem is in base and I think this PR should be merged this way, and then we can fix the base issue separately